pkgs/development/libraries/science/math/cudnn/generic.nix

```diff
@@ -1,8 +1,13 @@
 { stdenv
 , lib
-, cudatoolkit
+, zlib
+, cudaVersion
+, cudaMajorVersion
+, cudatoolkit # only used for .cc
+, libcublas
```

Comment:
this needs to have ? null for older cudatoolkit versions since the attribute does not exist then

Reply:
- `libcublas` now optional
- `cudatoolkit` not optional because using passthru `cc`
- cuda versions are present in cudaPackages_10 too

I've checked that all `cudaPackgaes_*.cudnn` evaluate into an `outPath`